benchmarks/benchmark_baselines.py

```diff
@@ -0,0 +1,33 @@
+"""Benchmarking for baselines."""
+import random
+
+from benchmarks import benchmark, iterate_experiments
+from benchmarks.experiments.baselines import continuous_mlp_baseline_tf
+from benchmarks.experiments.baselines import gaussian_cnn_baseline_tf
+from benchmarks.experiments.baselines import gaussian_mlp_baseline_tf
+
+
+@benchmark
+def ppo_continuous_mlp_baseline_tf_benchmarks():
+    """Run experiments for Continuous MLP Baseline TF benchmarking."""
+    seeds = random.sample(range(100), 3)
+    iterate_experiments(continuous_mlp_baseline_tf, seeds=seeds)
```

Comment:
can we make `env_ids` not optional for `iterate_experiments`? Having a default value seems not intuitive.